### ISSUE TITLE
Support for hardware timestamp for socketcan firmware

### ISSUE THREAD

**Issue by splatch** [UsbCANConverter-UCCbasic/UCCB_GS_Embedded#4]
SocketCAN allows to source hardware timestamp for all frames received through certain interface.

Currently using `candump .. -H` results in printing times of 1970 which means that this information is missing from controller.
Relevant code section in canutils:
https://github.com/linux-can/can-utils/blob/v2021.08.0/candump.c#L615L635

**Comment by splatch** [comment]
@Lukasz-Juranek Looking at `gs_usb` kernel driver itself I think this feature is already there. I see one of timestamp related constants referenced after switch to candle light. Did 0002 release include hardware timestamps?

**Comment by Lukasz-Juranek** [comment]
Yes at embeedded site it seems implemented but at GS_USB it is defined
in gs_usb_breq but value GS_USB_BREQ_TIMESTAMP is never send, so for me not
works with GS_USB

wt., 8 lut 2022 o 21:50 Łukasz Dywicki ***@***.***>
napisał(a):
